Ginger/GingerCore/NoSqlBase/GingerMongoDb.cs

```diff
@@ -264,14 +264,35 @@ public override void PerformDBAction()
                 switch (Action)
                 {
                     case Actions.ActDBValidation.eDBValidationType.FreeSQL:
-                        var result = collection.Find(GetQueryParamater(SQLCalculated, "find")).
+
+                        if (SQLCalculated.Contains("insertOne"))
+                        {
+                            BsonDocument insertDocumnet = BsonDocument.Parse(GetUpdateQueryParams(SQLCalculated));
+                            collection.InsertOne(insertDocumnet);
+                        }
+                        else if (SQLCalculated.Contains("insertMany"))
+                        {
+                            //need to find way to use insertMany()
+                            string queryParam = GetUpdateQueryParams(SQLCalculated).ToString();
```

Comment:
There is method insert many in collection. Why that can not be used instead of inserting one by one ?